integration/host_vulnerability_test.go

```diff
@@ -414,7 +414,10 @@ func validRFC3339(field string, jsonString string) bool {
 
 func machineIDExists(lw *api.Client) bool {
 	resp, err := lw.Vulnerabilities.Host.GetHostAssessment(machineID)
-	return err != nil || !resp.Ok
+	if err != nil {
```

Comment:
Wondering if it could be reduced to `return err == nil && resp.Ok` ?

Reply:
Could do, but I think the way it is now is a little more clear. We had a 1 liner here previously.

Reply:
Gonna leave as it for now